src/Ops.Web/Areas/Admin/Views/Features/FeatureDetails.cshtml

```diff
@@ -0,0 +1,181 @@
+﻿@model Ocuda.Ops.Controllers.Areas.Admin.ViewModels.Feature.FeatureViewModel
+
+@section styles {
+    <link rel="stylesheet"
+          href="~/devstyles/fontawesome-iconpicker/main.min.css"
```

Comment:
@madison-boman No references to `~/devstyles/` in the code, that is just for when running in development mode.